sam-app/hello_world/app.py

```diff
@@ -1,9 +1,152 @@
 import json
-
 import requests
 
+import boto3 #for the client
+import uuid  #for unique logStream names
+import time
 
-def lambda_handler(event, context):
+CONVERT_SECONDS_TO_MILLIS_FACTOR = 1000
+
+
+def validator(event):
```

Comment:
Don't use an if/else forest like this for validating the request. You should use JSONSchema instead. I'll send you an example.